common/lib/authentication/iam_authentication_plugin.ts

```diff
@@ -133,7 +134,8 @@ export class IamAuthenticationPlugin extends AbstractConnectionPlugin {
     }
   }
 
-  static clearCache(): void {
+  releaseResources(): Promise<void> {
```

Comment:
The okta and federated auth plugins still use clearCache, did we want to change those as well?